source/code-of-conduct/index.html.haml

```diff
@@ -1,44 +1,39 @@
 ---
-title: Brug - Code of Conduct - Ruby Belgium
+title: Code of Conduct - Ruby Belgium
 ---
 
-%header.header
+%header
```

Comment:
How is 

```css
section.alternate { background: grey; }
```
```html
<section class="alternate"></section>
```

more cumbersome compared to

```css
.section--alternate { background: grey; }
```
```html
<section class="section--alternate"></section>
```

Honest question. I really fail to see the downside in practical use. In which example would there be a clash using the first method?

In both cases, you will have 1 specific CSS rule for `section`, 1 for `li`, etc.

Also, `section--alternate` increases the specificity of the class name, so that's just transferring a concept that already exists in CSS to the class name.

And _if_ you need common rules for all `alternate`, you can define an `.alternate` CSS rule, inherited by all of the more specific ones. It's a CSS _feature_, not a troll spec designed to annoy the programmer and force her-him to invent an additional convention.